packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -94,18 +94,6 @@ describeNoCompat("Named root data stores", (getTestObjectProvider) => {
     const getRootDataStore = async (dataObject: ITestFluidObject, id: string, wait = true) =>
         runtimeOf(dataObject).getRootDataStore(id, wait);
 
-    const corruptedAPIAliasOp = async (runtime: IContainerRuntime, alias: string): Promise<boolean | Error> =>
```

Comment:
Did you mean to modify this file?

Reply:
Just some cleanup of this test (it relied on certain events from runtime). The test would fail with a timeout rather than any useful information.